woocommerce/woocommerce
# Migrate `@woocommerce/data` onboarding store to TS

### All Submissions:






### Changes proposed in this Pull Request:



Part of https://github.com/woocommerce/woocommerce/issues/33105

This PR migrates **onboarding** stores to TS and update type in ./client after migrating onboarding store to TS

### How to test the changes in this Pull Request:

1. Should run `pnpm exec turbo run build '--filter=@woocommerce/data'` without errors
3. Should run `pnpm run --filter @woocommerce/admin-library ts:check` without errors

### Other information:




### FOR PR REVIEWER ONLY:
